examples/slice/README.md

```diff
@@ -21,6 +21,7 @@ This folder contains example applications that showcase the IceRPC + Slice integ
 | [Retry](./Retry/)                          | Shows how to use the retry interceptor to retry failed requests.                                                                    |
 | [Secure](./Secure/)                        | Shows how to secure TCP connections with TLS.                                                                                       |
 | [Stream](./Stream/)                        | Shows how to stream data from a client to a server.                                                                                 |
+| [TcpFallback](./TcpFallback/)              | Shows how to create client and server applications that communicate over QUIC when possible and fall back to TCP.                   |
```

Comment:
At the risk of being _that_ guy: "fall back" is the action of falling back, "fallback" is a noun.
You "fall back" to your "fallback".

```suggestion
| [TcpFallback](./TcpFallback/)              | Shows how to create client and server applications that communicate over QUIC when possible but can fall back to TCP.                   |
```